config/svg/generateIllustrations.js

```diff
@@ -0,0 +1,55 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const glob = require('glob');
+const SVGO = require('svgo');
+const { generateComponent, generateIndex } = require('./generateSvgs');
+const svgoConfig = require('./config.illustration.json');
+
+const sourcePath = path.resolve(__dirname, '../../assets/illustrations');
+const destPath = path.resolve(__dirname, '../../src/components/Illustration/illustrations');
+const indexTemplatePath = path.resolve(__dirname, 'indexTemplate.ejs');
+const svgTemplatePath = path.resolve(__dirname, 'illustrationTemplate.ejs');
+
+const readFile = util.promisify(fs.readFile);
+const globFiles = util.promisify(glob);
+
+const optimizer = new SVGO(svgoConfig);
+
+function getSizeFromFilePath(filePath) {
+  const sizeRegex = /.+\/(\d+)\/.+\.svg/;
+  const sizeResults = filePath.match(sizeRegex);
+
+  if (sizeResults === null) {
+    console.error(`Could not determine svg size from path: ${filePath}`);
+  }
```

Comment:
Should you be returning or throwing here? Otherwise you're going to get a null reference exception in line 30.

Reply:
In the case that there is an invalid SVG path, I think we'll want to throw and stop the build. @swese44 and I verbally discussed this but forgot to translate it to code. I'll update.